Kickstarter-iOS/Features/PledgeView/Controllers/NoShippingPledgeViewController.swift

```diff
@@ -500,6 +506,52 @@ final class NoShippingPledgeViewController: UIViewController,
         self?.viewModel.inputs.scaFlowCompleted(with: status, error: error)
       }
   }
+
+  private func configurePledgeOverTimeUI(_ isPledgeOverTimeAllowed: Bool) {
+    guard isPledgeOverTimeAllowed else { return }
+
+    let collectionPlanStackView = UIStackView(frame: .zero)
+    collectionPlanStackView.axis = .vertical
+    collectionPlanStackView.spacing = Styles.grid(2)
+
+    let arrangedCollectionPlanSubviews = [
+      self.plansSectionLabel,
+      self.paymentPlansView
+    ]
+
+    arrangedCollectionPlanSubviews.forEach {
+      collectionPlanStackView.addArrangedSubview($0)
+    }
+
+    let collectionPlanSectionViews: [UIView] = [collectionPlanStackView]
```

Comment:
nit: use `collectionPlanStackView` in `arrangedInsetSubviews` directly since it's the only view in this list.